src/frontends/pytorch/src/op/sort.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+#define mark(...) context.mark_node(__VA_ARGS__)
+#define shared    std::make_shared
```

Comment:
Please remove these defines, I don't think this is a good practice

Reply:
Removed